src/services/arweave/SimpleArweaveDataProvider.ts

```diff
@@ -175,4 +175,16 @@ export class SimpleArweaveDataProvider implements ArweaveDataProvider {
       }
     }
   }
+
+  async arPrice(data: any): Promise<number> {
```

Comment:
nit: don't use any - either pass a buffer or the byte size directly

Reply:
changed